readme.md

```diff
@@ -0,0 +1,29 @@
+# Kiwi.com UI library
+
+## Table of Contents
+
+* [Motivation](#motivation)
+* [Repository Structure](#repository-structure)
+* [Contributing](#contributing)
+
+## Motivation
+
+The purpose of this repository is to open-source general UI components in use across Kiwi.com.
+
+It also serves as a testing ground for the best approach when it comes to CSS-in-JS. We have chosen [Styled components](https://www.styled-components.com) and [Styled JSX](https://github.com/zeit/styled-jsx) as the two front-runners. In time this repository will be changed to use only one technology. This discussion is aggregated here: https://github.com/kiwicom/react-kiwi-ui/issues/7.
+
+## Repository Structure
+
+This repository is broken up into 3 projects.
+
+| Project | Technology                                               | Description                     |
+| :------ | :------------------------------------------------------- | :------------------------------ |
+| BLKN    | [Styled JSX](https://github.com/zeit/styled-jsx)         | All Components for BLKN project |
+| Kiwicom | [Styled Components](https://www.styled-components.com)   | All Components for Kiwi.com     |
+| Icons   | [React-icons](https://gorangajic.github.io/react-icons/) | All Kiwi Icons                  |
```

Comment:
What? I was supposed to integrate react-icons somehow?

Reply:
I thought you were using those. Whatever tell me what to put there and I'll update it.

Reply:
No, the Frontend team is using react-icons on the Frontend project, but I thought I should add our custom icons. Now I'm even less sure what is the expected output of my current task...

Reply:
Yes... the task was to add our custom icons... So just tell me what I should put as the technologies used. I don't know what react-icons does and there was a discussion in person that mentioned this. Hence the assumption.

Reply:
`react-icons` takes SVG versions of material icons, font awesome etc. and makes them into components